docs/Documentation/Systems/Vermilion/running.md

```diff
@@ -346,370 +358,9 @@ make run
 
 
 ## Running VASP on Vermilion
+Please see the [VASP page](../../Applications/vasp.md) for detailed information and recommendations for running VASP on Vermilion. 
```

Comment:
Is this here in case users are looking for material they previously found here?
If so, okay to keep. Otherwise, no need to call out VASP specifically when all applications have their own pages.

Reply:
Yeah, put that there in case users are wondering where the info went. Can be removed eventually.